common/locales/en/contrib.json

```diff
@@ -54,8 +54,9 @@
     "playerTiers": "Player Tiers",
     "tier": "Tier",
     "visitHeroes": "Visit the Hall of Heroes (contributors and backers)",
-    "conLearn": "Learn more about contributor rewards",
+    "conLearn": "<a href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank'>Learn more about contributor rewards</a>",
```

Comment:
@MathWhiz Not sure.

@staff Any input?

Ideally, I think it should be consistent.  And if not consistent, just more accurately labeled for the sake of clarity.